javascript/ql/lib/semmle/javascript/frameworks/ClientRequests.qll

```diff
@@ -947,7 +947,7 @@ module ClientRequest {
   private class ClientRequestThreatModel extends ThreatModelSource::Range {
     ClientRequestThreatModel() { this = any(ClientRequest r).getAResponseDataNode() }
 
-    override string getThreatModel() { result = "response" }
+    override string getThreatModel() { result = "client-response" }
```

Comment:
```suggestion
    override string getThreatModel() { result = ["response", "browser-response"] }
```
To be forward compatible with the planned solution. I'll open a PR to make this a bit more accurate.